packages/server/src/common/index.ts

```diff
@@ -90,3 +103,4 @@ export * from './utils/layout-util';
 export * from './utils/logger';
 export * from './utils/promise-queue';
 export * from './utils/registry';
+
```

Comment:
nit: When I run `yarn generate:index` I only get a single new line in this file. Could you double check by running it on your end?